src/rest/SearchEvent.ts

```diff
@@ -1,19 +1,90 @@
 import { IAnalyticsEvent } from './AnalyticsEvent';
 import { IAnalyticsDynamicFacetMeta } from '../ui/Analytics/AnalyticsActionListMeta';
 
+/**
+ * Describes a Coveo Cloud usage analytics _search_ event.
+ */
 export interface ISearchEvent extends IAnalyticsEvent {
+  /**
+   * The unique identifier of the related query.
+   *
+   * **Note:** The framework normally sets this field by retrieving the information from the related Search API query response.
+   *
+   * **Example:** `74682726-0e20-46eb-85ac-f37259346f57`
+   */
   searchQueryUid: string;
+  /**
+   * The name of the query pipeline to which the related query was routed.
+   *
+   * **Note:** The framework normally sets this field by retrieving the information from the related Search API query response.
+   *
+   * **Example:** `PartnerPortalSearchPipeline`
+   */
   queryPipeline: string;
+  /**
+   * The name of the A/B test that applied to the related query.
+   *
+   * **Note:** This field may be set through the `splitTestRunName` option of the `Analytics` component. However, if the option is left undefined, the framework attempts to set this field by retrieving information from the related Search API query response.
+   *
+   * **Example:** `Test new ART model`
+   */
   splitTestRunName: string;
+  /**
+   * The version of the A/B test that applied to the related query (i.e., version A or version B).
+   *
+   * **Note:** This field may be set through the `splitTestRunVersion` option of the `Analytics` component. However, if the option is left undefined and the related Search API query response indicates that an A/B test was applied, the framework sets this field to the name of the query pipeline to which the query was routed.
+   *
+   * **Example:** `PartnerPortalSearchPipelineWithART`
+   */
   splitTestRunVersion: string;
-  mobile: boolean;
```

Comment:
The `mobile` and `responseTime` properties duplicated the IAnalyticsEvent interface properties of the same name, so I took the liberty to remove them from the ISearchEvent interface.